### ISSUE TITLE
REST API Documentation

### ISSUE THREAD

**Issue by carlospsikick** [Aiven-Open/karapace#181]
hello!
Is there a place where I could review the endpoints documentation? 
Similar to https://api.aiven.io/doc/#operation/ServiceKafkaTopicCreate

Thanks in advance

**Comment by lornajane** [comment]
Thanks for asking us about this, I've got to confess that the API docs are a work in progress but your nudge is welcome :) While we work on that, please let us know if you have any specific questions we can help with?

**Comment by mortenlj** [comment]
Since the REST part is supposed to be a near drop-in replacement for the Confluent REST proxy, is it safe to assume that the [Confluent docs](https://docs.confluent.io/platform/current/kafka-rest/index.html) will be useful to look at?

**Comment by lornajane** [comment]
Yes, the two products are diverging but pretty slowly so that should be a decent reference for anything that isn't super new on their side. I'll also leave this issue open so I remember to circle back with API docs once I have them available.

**Comment by OneCricketeer** [comment]
> drop-in replacement for the

Believe you meant Schema Registry here, which does expose an OpenApi specification

**Comment by Ashleycodesseo** [comment]
hey @lornajane I saw that this issue was still open. If you need any help with REST API documentation I'd be happy to contribute! Please let me know! Thanks!

**Comment by muralibasani** [comment]
@carlospsikick  I see meanwhile api reference docs are updated here. https://github.com/Aiven-Open/karapace?tab=readme-ov-file#schema-registry-api-reference

Pls open it, if you think it's not good enough. Closing this issue.
